tests/test_enterprise/api/test_views.py

```diff
@@ -105,6 +105,9 @@
 LICENSED_ENTERPISE_COURSE_ENROLLMENTS_REVOKE_ENDPOINT = reverse(
     'licensed-enterprise-course-enrollment-license-revoke'
 )
+EXPIRED_LICENSED_ENTERPISE_COURSE_ENROLLMENTS_ENDPOINT = reverse(
```

Comment:
nit: `EXPIRED_LICENSED_ENTERPISE_COURSE_ENROLLMENTS_ENDPOINT` is missing the second "R" in `ENTERPISE`